airflow/providers/datadog/hooks/datadog.py

```diff
@@ -115,8 +120,16 @@ def query_metric(self,
         return response
 
     # pylint: disable=too-many-arguments
-    def post_event(self, title, text, aggregation_key=None, alert_type=None, date_happened=None,
-                   handle=None, priority=None, related_event_id=None, tags=None, device_name=None):
+    def post_event(self, title: str,
+                   text: str,
+                   aggregation_key: Optional[str] = None,
+                   alert_type: Optional[str] = None,
+                   date_happened: Optional[int] = None,
+                   handle: Optional[str] = None,
+                   priority: Optional[str] = None,
+                   related_event_id: Optional[str] = None,
```

Comment:
`related_event_id` has a type of `id` (line 155 below) . Should this be a custom type  or is `str` correct?

Reply:
`related_event_id` is of type `int64` in the [Datadog API](https://docs.datadoghq.com/api/v1/events/#get-an-event). So in my opinion an `int` would be the proper choice here.

Reply:
> `related_event_id` is of type `int64` in the [Datadog API](https://docs.datadoghq.com/api/v1/events/#get-an-event). So in my opinion an `int` would be the proper choice here. What do you think?

I agree, but since the file referenced another type, i wanted to be sure. Thank you!